python/datasources/cdc_wisqars.py

```diff
@@ -249,108 +225,77 @@ def load_wisqars_df_from_data_dir(breakdown: str, geo_level: str):
             dtype={"Year": str},
         )
 
+        df.columns = df.columns.str.lower()
```

Comment:
I'm fine with this but why did you decide to lowercase the incoming columns and match against those?

Reply:
It's more of a preference. The shared `combine_race_and_ethnicity` function looks for a `ethnicity` column so I just opted to use lowercase column names for consistency.

Reply:
Ah gotcha. Seems like it might be worth changing the function API to require kwargs for the "race" and "ethnicity" column names? I feel like that's the normal
Pattern for our dataset util functions.